pages/day.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "A day on LiveWires"
+social_title: "technical activity week for 12–15s"
+social_image: "/assets/livewires-og.jpg"
+description: "LiveWires is a technical activity week for 12–15s. Start your summer learning about Jesus and technology."
+og_description: "Start your summer with a week learning about Jesus and technology."
+---
+
+# A Day On LiveWires
+
+<div class="columns" style="grid-template-columns: 1fr 2.2fr;">
+<img src="/assets/timetable.png" >
```

Comment:
Please can this have an `alt` attribute… such as…

`src="…" alt="a graphic showing the breakdown of a day into 15 minute increments"`